ccpp/data/GFS_typedefs.meta

```diff
@@ -5320,6 +5522,104 @@
   units = index
   dimensions = ()
   type = integer
+[mix_chem]
+  standard_name = rrfs_smoke_mynn_tracer_mixing
+  long_name = flag for rrfs smoke mynn tracer mixing
+  units = flag
+  dimensions = ()
+  type = logical
+  active = (flag_for_rrfs_smoke_coupling)
+[fire_turb]
+  standard_name = rrfs_smoke_mynn_enh_vermix
+  long_name = flag for rrfs smoke mynn enh vermix
+  units = flag
+  dimensions = ()
+  type = logical
+  active = (flag_for_rrfs_smoke_coupling)
+[seas_opt]
+  standard_name = rrfs_smoke_sea_salt_opt
+  long_name = rrfs smoke sea salt emission option
+  units = index
+  dimensions = ()
+  type = integer
+  active = (flag_for_rrfs_smoke_coupling)
+[dust_opt]
+  standard_name = rrfs_smoke_dust_opt
+  long_name = rrfs smoke dust chem option
+  units = index
+  dimensions = ()
+  type = integer
+  active = (flag_for_rrfs_smoke_coupling)
+[biomass_burn_opt]
+  standard_name = rrfs_smoke_biomass_burn_opt
+  long_name = rrfs smoke biomass burning option
+  units = index
+  dimensions = ()
+  type = integer
+  active = (flag_for_rrfs_smoke_coupling)
+[drydep_opt]
+  standard_name = rrfs_smoke_drydep_opt
+  long_name = rrfs smoke dry deposition option
+  units = index
+  dimensions = ()
+  type = integer
+  active = (flag_for_rrfs_smoke_coupling)
+[wetdep_ls_opt]
+  standard_name = rrfs_smoke_wetdep_ls_opt
+  long_name = rrfs smoke large scale wet deposition option
+  units = index
+  dimensions = ()
+  type = integer
+  active = (flag_for_rrfs_smoke_coupling)
+[do_plumerise]
+  standard_name = rrfs_smoke_plumerise_flag
+  long_name = rrfs smoke plumerise option
+  units = index
+  dimensions = ()
+  type = logical
+  active = (flag_for_rrfs_smoke_coupling)
+[plumerisefire_frq]
+  standard_name = rrfs_smoke_plumerisefire_frq
+  long_name = rrfs smoke add smoke option
+  units = index
+  dimensions = ()
+  type = integer
+  active = (flag_for_rrfs_smoke_coupling)
+[addsmoke_flag]
+  standard_name = rrfs_smoke_addsmoke_flag
+  long_name = rrfs smoke add smoke option
+  units = index
+  dimensions = ()
+  type = integer
+  active = (flag_for_rrfs_smoke_coupling)
+[smoke_forecast]
+  standard_name = rrfs_smoke_smoke_forecast_opt
+  long_name = flag for rrfs smoke forecast
+  units = flag
+  dimensions = ()
+  type = logical
+  active = (flag_for_rrfs_smoke_coupling)
+[aero_ind_fdb]
+  standard_name = rrfs_smoke_aero_ind_fdb_opt
```

Comment:
`do_smoke_aerosol_indirect_feedback`

Reply:
Are you suggesting this as the namelist variable name or the standard_name?

Reply:
All suggestions are standard names only, which means that any changes to metadata shouldn't need to be retested in RTs. Completing ccpp_prebuild.py successfully should suffice for testing.